FileMapper/copyFileAPI.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * API to copy a file from it’s source location to a destination location.
+ * 
+ * @param {StringArray} sourceFilePaths An array of Absolute local file paths of 
+ *                      the files which are needed to be copied.
+ * @param {String} targetFolderPath Absolute local folder path of the folder to which 
+ *                 the files are needed to be copied to.
+ */
+function copyFiles(sourceFilePaths, targetFolderPath) {
+  // First find the drive id for the target folder
+  var exists = hasFolder(targetFolderPath);
+  if(!exists){
```

Comment:
space after if  and closing braces -> if (!exists) {

Also in other places in this PR

Reply:
Done